docs/OV_Runtime_UG/migration_ov_2_0/deployment_migration.md

```diff
@@ -78,9 +78,9 @@ target_link_libraries(ov_c_app PRIVATE openvino::runtime::c)
 
 ### Native Interfaces
 
-To build applications without CMake interface, you can also use MSVC IDE, UNIX makefiles and any other interfaces, which have been changed as below:
+To build applications without the CMake interface, you can also use MSVC IDE, UNIX makefiles, and any other interface, which has been changed as shown here:
 
-**With Inference Engine of previous versions**:
+**With previous versions' Inference Engine**:
```

Comment:
I don't see the point of change here...

Reply:
Yep, nothing concrete there... Just thought it sounded better this way. I won't argue here if someone thinks it doesn't. But I would probably consider "the" in the original then.